moncli/types.py

```diff
@@ -211,4 +212,26 @@ def validate_timezone(self, value):
         try:
             pytz.timezone(value)
         except (UnknownTimeZoneError):
-            raise ValidationError('Unknown time zone "{}".'.format(value))
\ No newline at end of file
+            raise ValidationError('Unknown time zone "{}".'.format(value))
+
+class HourType(MondayType):
+    native_type = Hour
+    null_value = {}
+    allow_casts = (dict, )
+
+    def _cast(self, value):
+        try:
+            return Hour(value['hour'],value['minute'])
+        except KeyError:
+            raise ConversionError('Cannot convert value "{}" into Hour.'.format(value))
+    def _export(self, value):
+        return {'hour': value.hour, 'minute': value.minute}
+
+    def validate_hour(value):
```

Comment:
This is broken. As a class method, you need to pass the self in as the first parameter. 😠